doc/chapters/api/index.rst

```diff
@@ -0,0 +1,111 @@
+
+.. _api:
+
+API
+***
+
+This chapter documents the DBus and C API for interacting with the SoftwareContainerAgent. This is
+mostly useful for integrators.
+
+.. TODO::
+    This chapter is still to be written properly. Some bits and pieces have
+    been but they are under development.
+
+.. _dbus-api:
+
+DBus API
+========
+
+:Path: /com/pelagicore/SoftwareContainer
+:Interface: com.pelagicore.SoftwareContainerAgent
+
+Methods
+-------
+
+Ping
+----
+Ping()
```

Comment:
Not really tbh, but it's all a work in progress and just getting the stuff in there was enough to start with. I'm most likely not finishing this document in this story however as it's not part of what I should really be doing however, I was mostly adding it as a placeholder.